distribution/openrct2.d.ts

```diff
@@ -2676,7 +2669,7 @@ declare global {
         /**
          * The location and direction of where the car is on the track.
          */
-        trackLocation: CarTrackLocation;
+        trackLocation: CoordsXYZD;
```

Comment:
I think it's best to not revert this to the prior PR. Instead it'd be better to keep the getter as is, but only remove the setter as that's the broken one. Reading the vehicle's track type could still be useful in other scenarios.

Also make sure to make the properties on `CarTrackLocation` readonly as well, to avoid any confusion there. 🙂

```suggestion
        readonly trackLocation: CarTrackLocation;
```